types/underscore/index.d.ts

```diff
@@ -3710,9 +3711,13 @@ declare module _ {
         * @keys The key/value pairs to keep on `object`.
         * @return Copy of `object` with only the `keys` properties.
         **/
+        pick<T, K extends keyof T>(obj: T, ...keys: K[]): Pick<T, K>;
         pick(
             object: any,
             ...keys: any[]): any;
```

Comment:
In my opinion, if you make the strongly typed signatures completely cover all valid use cases of `pick`, which I think will be the case if you add both signatures from my previous comment, then this old, overly permissive signature should be removed.

Reply:
done